samples/core-projects/libraries/README.md

```diff
@@ -0,0 +1,83 @@
+# .NET Core Library Samples
+
+**Note:** These are compatibile with tooling that is still in active development.  Details are subject to change over time.
+
+These samples are buildable projects whose source are used for code snippets in [the guide for writing cross-platform libraries](../../../docs/libraries/libraries-with-cli.md).  They can be built and run using the .NET Core RC1 toolchain.
+
+They are small and intended to demonstrate how to target and build NuGet packages for different targets.
+
+To build/use any of these (using `new-library` as an example):
+
+1. Open your favorite Command Line Interface.
+
+2. Navigate to the top-level directory:
+
+	`$ cd new-library`
+
+3. Restore packages by typing the following:
+
+	`$ dnu restore -s "https://myget.org/F/xunit"`
+	
+	`$ dnu restore`
+	
+	Note: The first `dnu restore` is needed right not to allow for unit tests to run.  This will not be required in the future.
+
+4. To build and package the library as a NuGet package, type the following:
+
+	`$ cd src/Library`
+	
+	`$ dnu build`
+	
+	`$ dnu pack`
+	
+	Check out the `/bin/Debug` directory to see the generated artifacts and `.nupkg`.
+
+5. To run unit tests (only applicable to `new-library`):
+
+	`$ cd ../../test/LibraryTests`
+	
+	`$ dnx test`
+
+And that's it!
+
+## new-library
+
+The project under `/new-library` is a project targeting **only** .NET Core.  Backwards compatibility is not a goal.
+
+It also showcases two other things: how to use multiple projects and how to test.
+
+There are two libraries showcased.  The first, `DependencyLibrary`, contains functionality that the second, `Library` uses.  The `LibraryTests` test project takes a dependency on `Library` to test that project.
+
+## frameworks-library
+
+**IMPORTANT:** This project requies Windows and .NET Framework installed on your machine.
+
+The project under `/frameworks-library` demonstrates a project you can use the CLI tooling for to target .NET Framework.  It does so with a simple project targeting .NET Framework 4.0 and another project targeting .NET Framework versions 4.0, 4.5, 4.5.1, 4.5.2, 4.6, and 4.6.1.
```

Comment:
I'll have to think about this a bit.  These only really exist so that they can be embedded in the article.  I'm tentatively thinking that I'll remove the second project.